docs/UserGuide.md

```diff
@@ -361,7 +361,7 @@ Exiting Calendar...
 ```
 The application will exit after displaying a goodbye message.
 
-### ICS Exporting and Importing: `ics` `[coming in v2.1]`
+### ICS Exporting and Importing: `ics` `[coming in future versions]`
```

Comment:
Good spot on this! We should definitely leave the ICS handling to future versions :)